src/main/java/io/xstefank/wildfly/bot/LifecycleProcessor.java

```diff
@@ -46,8 +57,29 @@ void onStart(@Observes StartupEvent event) {
                 GitHub app = clientProvider.getInstallationClient(installation.getId());
                 for (GHRepository repository : app.getInstallation().listRepositories()) {
                     try {
-                        fileProvider.fetchConfigFile(repository, RuntimeConstants.CONFIG_FILE_NAME, ConfigFile.Source.DEFAULT, WildFlyConfigFile.class).get();
-                        LOG.infof("The configuration file from the repository %s was parsed successfully.", repository.getFullName());
+                        WildFlyConfigFile wildflyBotConfigFile = fileProvider.fetchConfigFile(repository, RuntimeConstants.CONFIG_FILE_NAME, ConfigFile.Source.DEFAULT, WildFlyConfigFile.class).get();
+                        List<String> emailAddresses = wildflyBotConfigFile.wildfly.emails;
+                        List<String> problems = configFileChangeProcessor.validateFile(wildflyBotConfigFile);
+                        if (problems.isEmpty()) {
+                            LOG.infof("The configuration file from the repository %s was parsed successfully.", repository.getFullName());
+                        } else {
+                            LOG.errorf("The configuration file from the repository %s was not parsed successfully due to following problems: %s", repository.getFullName(), problems);
+
+                            if (username.isPresent() && emailAddresses != null && !emailAddresses.isEmpty()) {
+                                LOG.infof("Sending email to the following emails [%s].", String.join(", ", emailAddresses));
+                                mailer.send(
+                                        new Mail()
+                                                .setSubject("Unsuccessful installation of Wildfly Bot Application")
+                                                .setText(String.format("""
+                                                        Hello,\n
```

Comment:
This will add two new lines, are you aware of that?

Reply:
Yes, I didn't want to use `\n` at all, but I had to because of the `maven-checkstyle-plugin` didn't accept empty lines in string